assessments/0002-notary-project.md

```diff
@@ -0,0 +1,176 @@
+
+**New user content**:
+- What is there does a good job of explaining the project, but doesn't provide many docs focused on new users.
+
+**Content maintainability**:
+- The docs currently aren't versioned or localized, and is depending on GitHub for search.
+
+**Content creation processes**:
+- There are links to community discussions, but as there is no website yet there is no space to point to regarding docs creation.
+
+### Recommendations
+
+V1 Docs will likely need to be reworked once an information architecture is setup for the new site. The new site IA can likely be informed by the existing V1 docs.
+
+The notaryproject.dev site should primarily focus on the V2 work, but V1 needs to be represented as well.
+- V1 content seems quite good, but looks like it was developed for a site that no longer uses it.
+- For MVP, I'd suggest focusing on V2 content, and linking out to the existing V1 documentation in the location that is currently rather than incorporating it right off the bat.
+
+Eventually I'd expect that V1 would be moved to the Notary Project org, but that is a project level consideration. Even if the whole repo doesn't get moved, there may be value in moving the docs out of TUF's notary repo and into the notaryproject/notaryproject.dev repo.
+
+## Contributor documentation 
+
+**Note**, as my recommendation above is to link to the V1 documentation as is for the website MVP, the remainder of this assessment will only be looking at the existing V2 documentation.
+
+Criteria 
+- 1 = (Is not present or requires significant work)
+- 3 = (is present, but needs work)
+- 5 = (is executed extremely well or no improvement required) |
+
+| Criteria                                  | 1   | 2   | 3   | 4   | 5   |
+| ---                                       | --- | --- | --- | --- | --- |
+| Communication methods documented          |     |     | ✅  |     |     |
+| Beginner friendly issue backlog           |     | ✅  |     |     |     |
+| “New contributor” getting started content | ✅  |     |     |     |     |
+| Project governance documentation          | ✅  |     |     |     |     |
+
+### Comments
+
+_Provide comments for each rating above, 1-2 sentences max, bullet point list_
+
+**Communication methods documented**
+- There is a Contributing & Conversations section in the main README (tho, it is mis-linked in the table of contents)
+
+**Beginner friendly issue backlog**
+- There are "Documentation," "spec," and "good first issue" issue tags
+- Most issues seem to link out to hackmd.io notes, but those note are are well written and informative
+
+**“New contributor” getting started content**
+- Same as above, there is a Contributing & Conversations section, but there are not yet any instructions for contributions.
+
+**Project governance documentation**
+- This appears to be actively being worked on, and there is a "Governance" issue tag in use. 
+
+
+### Recommendations
```

Comment:
Given the large gaps in project governance and contributor documentation, one of my major recommendations both in this second *and* for the project as a whole is that they sort that out ASAP. I would mention that a lack of contributor documentation and formal governance will be a huge roadblock to their development as a project and their ability to attract new contributors will be hindered as a result. 

I'm confused as to why this didn't show up as a recommendation for you.